docs/src/modules/javascript/pages/views.adoc

```diff
@@ -4,18 +4,28 @@ include::ROOT:partial$include.adoc[]
 include::partial$views.adoc[]
 
 [#value-entity]
-== View from Value Entity
+== View from a Value Entity
 
-Using an example of a customer registry, you can define a `Customer` in Protobuf as:
+Consider an example of a Customer Registry service with a `customer` Value Entity. When `customer` state changes, the entire state is emitted as a value change. Value changes update any associated Views. To create a View that shows customers by their name:
+
+. <<_define_the_view_service_descriptor>> for a service that selects customers by name and associates a table name with the View. The table is created and used by Akka Serverless to store the View, use any name for the table.
+. <<#register-view>>.
+
+This example assumes the following `customer` state is defined in a `customer_domain.proto` file:
 
 [source,proto,indent=0]
 ----
 include::example$js-customer-registry/customer_domain.proto[tag=domain]
 ----
 
-When `Customer` state changes, the entire state is emitted as a value change, which updates any associated Views.
+=== Define the View service descriptor
 
-To get a view of multiple customers by their name, define the View as a `service` in Protobuf:
+To get a view of multiple customers by their name, define the View as a `service` in Protobuf. The descriptor defines:
+
+* How to update the View
+* The source of View data
+* A `table` property that can be any name
```

Comment:
It may be useful to say more than just that the table can be given any name. The main point here is that you refer to the given name for the table in queries. So if you have `table: "customers"` then your queries have `SELECT * FROM customers`.

Reply:
See https://github.com/lightbend/akkaserverless-javascript-sdk/pull/139/files